.repositories.yaml

```diff
@@ -8,20 +8,20 @@
 # List of repositories of Eclipse Che components
 ---
 repositories:
-  - url: https://github.com/eclipse/che
+  - url: https://github.com/eclipse-che/che-server/
```

Comment:
here we still need to have both che and che-server (to track PR on both repositories)